docs/uberon-release.md

```diff
@@ -0,0 +1,76 @@
+# Uberon Release workflow
+
+While Uberon is an ODK ontology, it has a specific workflow for releases due to its large size and the limitations on standard GitHub releases.
+
+## Requirements
+
+Aside from the standard requirements needed for ODK workflow, `GH` is required. 
+Instructions on how to install `GH` can be found [here](https://cli.github.com/manual/installation)
+
+You will need to log in to your GitHub account on `GH` before you can uses it. To do this, enter the following in your terminal:
+```
+gh auth login
+```
+
+You can then follow instructions below for web browser login (or use your prefer means of logging in).
+```
+% gh auth login
+? What account do you want to log into? GitHub.com
+? What is your preferred protocol for Git operations? SSH
+? Generate a new SSH key to add to your GitHub account? Yes
+? Enter a passphrase for your new SSH key (Optional) 
+? Title for your SSH key: GitHub SSH
+? How would you like to authenticate GitHub CLI? Login with a web browser
+
+! First copy your one-time code: XXXX-XXXX
+Press Enter to open github.com in your browser... 
+✓ Authentication complete.
+- gh config set -h github.com git_protocol ssh
+✓ Configured git protocol
+✓ Uploaded the SSH key to your GitHub account: /Users/username/.ssh/id_ed25519.pub
+✓ Logged in as username
+```
+
+## Release Process
+
+### Preparation 
+
+Preparation:
+
+1. Ensure that all your pull requests are merged into your main (master) branch
+1. Make sure that all changes to master are committed to Github (`git status` should say that there are no modified files)
+1. Locally make sure you have the latest changes from master (`git pull`)
+1. Checkout a new branch (e.g. `git checkout -b release-2021-01-01`)
+1. You may or may not want to refresh your imports as part of your release strategy (see [here](UpdateImports.md))(Note: in UBERON we decouple our imports and releases - we hence advice that you do not update imports)
+1. Make sure you have the latest ODK installed by running `docker pull obolibrary/odkfull`
+
+To actually run the release, you:
+
+1. Open a command line terminal window and navigate to the src/ontology directory (`cd uberon/src/ontology`)
+1. Run the release using `sh run.sh make IMP=false uberon DEPLOY_GH=false`. This will build all files and copy them to the correct place. (Note: the `IMP=false` is used to decouple imports refresh with release)
```

Comment:
@matentzn I've explicitly put IMP=false to decouple imports with release, just wanted to make sure you are ok with this being in docs - figured given its internal uberon docs we can do this

Reply:
No need - `uberon` already does that. I would remove it. Just `sh run.sh make uberon DEPLOY_GH=false`.